openquake/calculators/export/__init__.py

```diff
@@ -82,7 +82,7 @@
 }
 
 
-FIELD_DESCRIPTION = {
+LOSS_FIELD_DESCRIPTION = {
```

Comment:
I prefer `AGGRISK_FIELD_DESCRIPTION` since it will not be a loss when we will implement scenario damages.